### ISSUE TITLE
Panic when running report command with --yesterday flag

### ISSUE THREAD

**Issue by hovsater** [jotaen/klog#221]
```console
$  klog report -fd --yesterday @jun
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/jotaen/klog/klog/app/cli.(*Report).Run(0x1400026a170, {0x100bed588, 0x140002a0400})
	/home/runner/work/klog/klog/klog/app/cli/report.go:46 +0x94c
reflect.Value.call({0x100ba9ee0?, 0x1400026a170?, 0x14000139868?}, {0x100aa3bb9, 0x4}, {0x140003d0c48, 0x1, 0x10091a2a8?})
	/opt/hostedtoolcache/go/1.18.5/x64/src/reflect/value.go:556 +0x5e4
reflect.Value.Call({0x100ba9ee0?, 0x1400026a170?, 0x100b93300?}, {0x140003d0c48, 0x1, 0x1})
	/opt/hostedtoolcache/go/1.18.5/x64/src/reflect/value.go:339 +0x98
github.com/alecthomas/kong.callMethod({0x100aa39f7, 0x3}, {0x100bd1500?, 0x1400026a170?, 0x3?}, {0x100ba9ee0?, 0x1400026a170?, 0x0?}, 0x14000139aa8?)
	/home/runner/go/pkg/mod/github.com/alecthomas/kong@v0.2.22/callbacks.go:97 +0x3a0
github.com/alecthomas/kong.(*Context).RunNode(0x140002a0600, 0x140002782a0, {0x0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/alecthomas/kong@v0.2.22/context.go:731 +0x468
github.com/alecthomas/kong.(*Context).Run(0x140002346c0?, {0x0?, 0x140002a0400?, 0x100b65040?})
	/home/runner/go/pkg/mod/github.com/alecthomas/kong@v0.2.22/context.go:756 +0x148
github.com/jotaen/klog/klog/app/cli/main.Run({0x1400001604d, 0xc}, {{0x100ac2cc0, 0x3662}, {0x100abcc40, 0x442}, {0x100abd0a0, 0xba7}, {0x100b42fb0, 0x4}, ...}, ...)
	/home/runner/work/klog/klog/klog/app/cli/main/cli.go:78 +0x400
main.main()
	/home/runner/work/klog/klog/klog.go:38 +0x1fc
```

Yesterday's date look like this:

```
2022-10-28 (8h!)
    8h
```

**Comment by jotaen** [comment]
Oh, thanks for reporting this. It’s fixed in v5.3: https://github.com/jotaen/klog/releases/tag/v5.3

**Comment by hovsater** [comment]
Thank you!
